browser/js/play/gameCanvas.js

```diff
@@ -28,7 +28,7 @@ app.directive('gameCanvas', function($window, $injector) {
   };
 
   return {
-    template: '<div ng-class="currentBomb ? \'blurry\' : \'not-blurry\'" id=\'playGame\' balls></div><bomb-view ng-if="currentBomb" ></bomb-view>',
+    template: '<div ng-class="currentBomb || saveScore ? \'blurry\' : \'not-blurry\'" id=\'playGame\' balls></div><bomb-view ng-if="currentBomb" ></bomb-view><save-score ng-if="saveScore" ></save-score>',
```

Comment:
This is kind of hard to read, maybe throw some newlines in there? like: 
 template: '<div ng-class="currentBomb || saveScore ? \'blurry\' : \'not-blurry\'" id=\'playGame\' balls></div>
<bomb-view ng-if="currentBomb" ></bomb-view>
<save-score ng-if="saveScore" ></save-score>',